config/routes.rb

```diff
@@ -7,4 +7,5 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  resources :users
```

Comment:
Idiomatic

ルーティングを制限しよう。
このままだと、updateやdestroyも定義されてしまうので。

```suggestion
  resources :users, only: %i[show index]
```

Ref: https://railsguides.jp/routing.html#%E4%BD%9C%E6%88%90%E3%81%95%E3%82%8C%E3%82%8B%E3%83%AB%E3%83%BC%E3%83%86%E3%82%A3%E3%83%B3%E3%82%B0%E3%82%92%E5%88%B6%E9%99%90%E3%81%99%E3%82%8B

Reply:
[567636b](https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/26/commits/567636bbe2d51e5d4c56a232c15e934a9ee5fadf) で対応しました。